CircuitPython_Simple_Wordclock/code.py

```diff
@@ -0,0 +1,139 @@
+# Vertical Word Clock using the Adafruit Feather M4 and
+#   the Adafruit DS3231 real-time clock FeatherWing
+
+import time
+import board
+import busio as io
+import digitalio
+import adafruit_ds3231
+import neopixel
+
+i2c = io.I2C(board.SCL, board.SDA)
+
+# Create the RTC instance:
+rtc = adafruit_ds3231.DS3231(i2c)
+
+LED13 = digitalio.DigitalInOut(board.D13)
+LED13.direction = digitalio.Direction.OUTPUT
+
+pixel_pin = board.D5
+num_pixels = 21
+pixels = neopixel.NeoPixel(pixel_pin, num_pixels, brightness=0.8)
+pixels.fill((0, 0, 0))
+COLOR = (0, 200, 0)  # Green 
+
+# Bitmap values for each value. These can be OR'ed together
+THREE = 0b00000000000000000001
```

Comment:
these 0b consts are a little hard to read, please replace with `1 << 0` thru `1 << 15` (or whatever)

Reply:
They were very easy to see before Mu didn't like the spaces so they lined up showing the bit pattern.

Reply:
and hard for people to change around the bits - we dont need to save any space here :)

Reply:
roger, changed